Sources/Filters/General/AppendPolyData/test/testAppendPolyData.js

```diff
@@ -66,17 +66,16 @@ test('Test vtkAppendPolyData execution', (t) => {
 });
 
 test('Test addInputData edge case', (t) => {
-  const gc = testUtils.createGarbageCollector(t);
-  const appender = gc.registerResource(vtkAppendPolyData.newInstance());
```

Comment:
Why do you remove the GC in that test?

Reply:
It is not an asynchronous test, hence there is no need for GC. Did I miss something ?

Reply:
Are you saying ALL tests must use testUtils GarbageCollector ?

Reply:
Yes otherwise, the canvas stay around (as nobody delete them) and we run out of available one.  The gc in test help to clear things between tests.

Reply:
I mean the rendering ones. Having leaks in test with objects is not a big deal, but gl canvas are limited to 16 or something like that.